code/game/turfs/space/space.dm

```diff
@@ -16,7 +16,7 @@
 
 
 /turf/open/space/New()
-	update_icon()
+	icon_state = SPACE_ICON_STATE
```

Comment:
To answer this question because I'm browsing thru my changes to re-do them and because I get a sick pleasure out of proving you wrong chnkr:

Turf space intentionally doesn't call ..() in New(). Yes, I looked at doing just that, I had that exact same thought, but there are already New()s in open and turf, so this 1 proc call prevents 2 proc calls, and removing it would be a net 1 proc call gained per turf.